.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,39 @@
+name: Nightly
+on:
+  schedule:
```

Comment:
nice-to-have: also run this workflow on changes to this file (we use paths-filter for this in the monorepo's `pr.yaml`)